src/auth/auth_authentik_openid.py

```diff
@@ -0,0 +1,54 @@
+import logging
+
+from tornado import escape
+
+from auth.auth_abstract_oauth import AbstractOauthAuthenticator, _OauthUserInfo
+from model import model_helper
+
+LOGGER = logging.getLogger('script_server.GoogleOauthAuthorizer')
+
+
+# noinspection PyProtectedMember
+class AuthentikOpenidAuthenticator(AbstractOauthAuthenticator):
+    def __init__(self, params_dict):
+        authenitk_url = model_helper.read_obligatory(
+            params_dict,
+            'authenitk_url',
+            ': should contain openid url, e.g. http://localhost:9001/')
+        if not authenitk_url.endswith('/'):
+            authenitk_url = authenitk_url + '/'
+        self._authenitk_url = authenitk_url
+
+        # (oauth_authorize_url, oauth_token_url, oauth_scope, params_dict):
+        super().__init__(authenitk_url + 'application/o/authorize/',
+                         authenitk_url + 'application/o/token/',
+                         # "openid" scope is needed since version 20:
```

Comment:
I think this comment is only relevant for Keycloak. Not sure about profiles below